docs/Makefile

```diff
@@ -6,3 +6,8 @@ MAKEFLAGS += --warn-undefined-variables
 MAKEFLAGS += --no-builtin-rule
 
 include docs.mk
+
+$(REPOS_PATH)/mimir-$(MIMIR_DOCS_VERSION):
+	git worktree add -f $@ origin/$(MIMIR_DOCS_BRANCH)
```

Comment:
According to the comment in `make-docs` REPOS_PATH can be ":" separated list. So I'd use GIT_ROOT/.. instead and add that to REPOS_PATH or something like that.

Reply:
@jdbaldry, what is the next action here?

Reply:
Thanks for catching, resolved in [e156af8](https://github.com/grafana/mimir/pull/5716/commits/e156af81dcd93284b57c6a50e9fb661792acdbe7)